certbot/docs/install.rst

```diff
@@ -6,80 +6,49 @@ Get Certbot
    :local:
 
 
-About Certbot
-=============
 
-*Certbot is meant to be run directly on a web server*, normally by a system administrator. In most cases, running Certbot on your personal computer is not a useful option. The instructions below relate to installing and running Certbot on a server.
+.. _system_requirements:
 
-System administrators can use Certbot directly to request certificates; they should *not* allow unprivileged users to run arbitrary Certbot commands as ``root``, because Certbot allows its user to specify arbitrary file locations and run arbitrary scripts.
+System Requirements
+===================
 
-Certbot is packaged for many common operating systems and web servers. Check whether
-``certbot`` (or ``letsencrypt``) is packaged for your web server's OS by visiting
-certbot.eff.org_, where you will also find the correct installation instructions for
-your system.
+- Python 3.7+
```

Comment:
For most installation methods, Python is not required to be installed separately. I think only our packagers and `pip` users care about this. For the latter, the requirement is documented in the instruction generator.

```suggestion
```